### PR TITLE
Android/ARM64: fixed incorrect path of libs/obj

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to update correctly the existing path of libs and obj folder
after improving the configuration file to generate a C/C++ library.

**Changelog**
1. Fixed indentation
2. Replaced ../{libs,obj} with ./{libs,obj}

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

File: jni/Android-nnstreamer.mk [hunk 1]
```diff
@@ -83,10 +83,10 @@ $(foreach item,$(so_names_gst),$(eval $(call shared_lib_gst,$(item))))
...
 
 # Please keep the pthread and openmp library for checking a compatibility
-LOCAL_ARM_NEON := true
-LOCAL_CFLAGS   += -O0 -DVERSION=\"$(NNSTREAMER_VERSION)\"
-LOCAL_CXXFLAGS += -std=c++11
-LOCAL_CFLAGS   += -pthread -fopenmp
+LOCAL_ARM_NEON      := true
+LOCAL_CFLAGS        += -O0 -DVERSION=\"$(NNSTREAMER_VERSION)\"
+LOCAL_CXXFLAGS      += -std=c++11
+LOCAL_CFLAGS        += -pthread -fopenmp
```
Comment (myungjoo) [inline jni/Android-nnstreamer.mk]: unrelated changes?

Comment (leemgs) [reply jni/Android-nnstreamer.mk]: > 1. Fixed indentation
Need to be changed to keep same location with another statements.
